Source/DafnyStandardLibraries/src/Std/Strings.dfy

```diff
@@ -172,6 +178,9 @@ module {:disableNonlinearArithmetic} Std.Strings {
         'a' := 0xA, 'b' := 0xB, 'c' := 0xC, 'd' := 0xD, 'e' := 0xE, 'f' := 0xF,
         'A' := 0xA, 'B' := 0xB, 'C' := 0xC, 'D' := 0xD, 'E' := 0xE, 'F' := 0xF
       ]
+    // The size of the map makes this impractical to verify easily.
```

Comment:
I find this fascinating: I couldn't find an `{:rlimit}` value high enough to keep this from timing out without hitting an overflow exception in Boogie. :) And yet it's totally tractable to just statically evaluate the ensures clause, since there are so few values to test, and this is arguably just as sound a proof as verification!